cset-workflow/meta/rose-meta.conf

```
@@ -546,7 +546,7 @@ sort-key=pressure3
 
 [template variables=LFRIC_MODEL_LEVEL_MODEL_FIELDS]
 ns=Diagnostics
-description=List of standard names of model fields on model levels to plot.
+description=List of standard names of model fields on full_levels or half_levels to plot.
```

```suggestion
description=List of standard names of model fields on full_levels to plot.
```
This recipe only checks the single coordinate.